src/azure/storage/config.rs

```diff
@@ -44,4 +44,22 @@ pub struct Config {
     ///
     /// This is part of use AAD(Azure Active Directory) authenticate on Azure VM
     pub endpoint: Option<String>,
+    /// `azure_federated_token` value will be loaded from:
+    ///
+    /// - this field if it's `is_some`
+    /// - env value: [`AZURE_FEDERATED_TOKEN`]
+    /// - profile config: `azure_federated_token_file`
+    pub azure_federated_token: Option<String>,
+    /// `azure_federated_token_file` value will be loaded from:
+    ///
+    /// - this field if it's `is_some`
+    /// - env value: [`AZURE_FEDERATED_TOKEN_FILE`]
+    /// - profile config: `azure_federated_token_file`
+    pub azure_federated_token_file: Option<String>,
```

Comment:
We can just store `federated_token` inside.

Reply:
What if user only set `federated_token_file` ? I saw in azure-sdk-rust, they check both `federated_token`  and `federated_token_file`, refer to https://github.com/Azure/azure-sdk-for-rust/blob/22ab1128f183fc80b97bb53b15486cc60702b61c/sdk/identity/src/token_credentials/workload_identity_credentials.rs#L92.

Reply:
> What if user only set `federated_token_file` ? I saw in azure-sdk-rust, they check both `federated_token` and `federated_token_file`, refer to [Azure/azure-sdk-for-rust@`22ab112`/sdk/identity/src/token_credentials/workload_identity_credentials.rs#L92](https://github.com/Azure/azure-sdk-for-rust/blob/22ab1128f183fc80b97bb53b15486cc60702b61c/sdk/identity/src/token_credentials/workload_identity_credentials.rs#L92).

We can support loading from both `federated_token` and `federated_token_file` but only store it's content.